src/tests/ngraph_helpers/ngraph_functions/include/ngraph_functions/node_builders/param_vector.hpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "openvino/op/parameter.hpp"
+
+namespace ov {
+namespace test {
+namespace utils {
+namespace builder {
```

Comment:
Let's reduce number of layers in namespaces
```suggestion
namespace builder {
```

Reply:
`ov::test::builder` will conflict with `ngraph::builder` because a lot of files contains simultaneous `using namespace` for `ov::test` and `ngraph`

We can do this when `ngraph::builder` is removed